pkg/kando/process_client.go

```diff
@@ -42,3 +46,11 @@ func processAsJSONFlagValue(cmd *cobra.Command) bool {
 	}
 	return b
 }
+
+func processSignalIdentifierFlagValue(cmd *cobra.Command) int {
+	q, err := cmd.Flags().GetInt(processSignalIdentifierFlagName)
```

Comment:
If this parameter is required, why not make it a command line argument? If there's a parameter arguement, we should return an error with usage rather than panic which can be hard to hard to understand.

Reply:
Fair point.  Ill add it as an argument.